Twittimation/Commands/SetMaxLikes.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using Twittimation.IO;
+
+namespace Twittimation.Commands
+{
+    public sealed class SetMaxLikes : Command
```

Comment:
This should be per Followed Account. So, the two args needs are `Amount` and `AccountUsername`, preferably in `@user` form